app/classes/auto_complete.rb

```diff
@@ -100,7 +100,7 @@ def refine_matches
         return used + part if matches.length <= limit
       end
       if n < words.length
-        used += word + " "
+        used += "#{word} "
```

Comment:
I'm really surprised rubocop doesn't require: used = "#{used}#{word} ".  Likewise above used + part... I guess it just doesn't know that used is a String?  Not that I'm complaining, mind you!  I think the string interpolation version of concatenation is ugly...